ietf/api/tests_ietf_utils.py

```diff
@@ -31,16 +44,38 @@ def protected_function(request):
         result = protected_function(request)
         self.assertEqual(result, "Access granted: GET")
 
-        # request with an valid token
+        # request with an invalid token
         request = RequestFactory().get(
             "/some/url", headers={"X_API_KEY": "invalid-token"}
         )
         result = protected_function(request)
         self.assertEqual(result.status_code, 403)
 
+        # request without a token
+        request = RequestFactory().get("/some/url", headers={"X_API_KEY": ""})
```

Comment:
This tests a blank header, but I think we need the no-header-at-all case.

Reply:
4e80ea1f900bd44a14493a7272449acf0e73b31a should address this.